README.md

```diff
@@ -101,10 +101,24 @@ function YourComponent() {
 
 ### Library defaults
 
-This library provides some default styles. These default styles can be used as a foundation to build on top of or just as a reference for you to replace completely. All default styles in this library are scoped within the `.cio-autocomplete` css selector.
+By default, importing react components or hooks from this library does not pull any css into your project.
 
- - If you would like to remove all default styling, simply pass an empty string or your own custom autocomplete container className as the value for the `autocompleteClassName` option
- - If you would like to layer your own custom styles on top of the library default styles, you can do so by passing additional className(s) of your choosing `autocompleteClassName='cio-autocomplete custom-autocomplete-container'`
+If you wish to use some starter styles from this library, add an import statement similar to the example import statement below:
+
+```js
+import 'node_modules/@constructor-io/constructorio-ui-autocomplete/lib/mjs/components/Autocomplete/Autocomplete.css';
```

Comment:
Not for this PR, but we might look into putting the CSS somewhere more accessible. Since it's not imported directly from the component anymore, we have more freedom and it's pretty easy to do. 
Something like `node_modules/@constructor-io/constructorio-ui-autocomplete/Autocomplete.css`
Also I didn't check this but does it work if you drop the `node_modules` part? `@constructor-io/constructorio-ui-autocomplete....`

Reply:
Ohhh yeah I'd love if we could make `@constructor-io/constructorio-ui-autocomplete/Autocomplete.css` work!

> when testing out with sales-demos repo, current patterns only worked for me when I included the `node_modules` at the beginning... not what I would have expected 🤷🏻